pandas/tests/indexes/datetimes/test_ops.py

```diff
@@ -47,6 +47,27 @@ def test_ops_properties_basic(self):
         assert s.day == 10
         pytest.raises(AttributeError, lambda: s.weekday)
 
+    # TODO: figure out where in tests.reductions this belongs
+    def test_mean(self, tz_naive_fixture):
```

Comment:
seems *very* duplicative of the test below, is there a reason?

Reply:
bc at the time when this was written Index reduction tests had not been collected in tests.reductions.  Will move/de-duplicate.